src/components/articles/Article.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import '../../styles/Article.css';
+import Button from "../../atoms/Button";
+
+const Article = ({ imgSrc, title, author, tag, description, fullStoryLink }) => (
+    <article>
```

Comment:
Para que no haya problemas con los estilios si alguna vez se usa <article> en otra parte mejor agregar una clase especifica.
```suggestion
    <article className="article-wrapper">
```